sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/physical/partitioning.scala

```diff
@@ -626,6 +626,47 @@ case class BroadcastPartitioning(mode: BroadcastMode) extends Partitioning {
  *   - Creating a partitioning that can be used to re-partition another child, so that to make it
  *      having a compatible partitioning as this node.
  */
+
```

Comment:
This is really late comment, but just noticed that this now orphans the above comment from its original location (trait ShuffleSpec)

Reply:
ah good catch!

@shujingyang-db please help fix it, thanks!